validator/src/main.rs

```diff
@@ -1210,19 +1210,6 @@ pub fn main() {
                       will not push/pull from from validators outside this set. \
                       [default: all validators]")
         )
-        .arg(
-            Arg::with_name("no_rocksdb_compaction")
```

Comment:
i think it's time to remove these flags finally, even at the cost of some command line compatility churn.

so, we need to announce this and following some flags to be stripped from all scripts well in advance. After all, these flags is no-op as of now. (cc: @willhickey; this pr won't be merged soon. just very early headsup)